modules/series-service-impl/src/main/java/org/opencastproject/series/endpoint/SeriesRestService.java

```diff
@@ -1737,6 +1737,55 @@ public Response getSeriesElement(
     }
   }
 
+  @PUT
+  @Path("{seriesId}/extended-metadata/{type}")
```

Comment:
Seems a bit weird to have a way to set extended metadata without a way to get it. Might be reasonable to add a @GET endpoint?

Reply:
You can actually already get extended metadata via GET /{seriesId}/elements/{elementType}". And the new endpoint is actually just a wrapper for PUT /{seriesId}/elements/{elementType} as well. I intended to get rid of the old endpoints, but then that fell of my radar, not sure if/when that's going to happen now. But for now, this doesn't break the old endpoints, even if they're not explicitly stable.